.env.local.example

```diff
@@ -1,14 +0,0 @@
-# nextjs prioritises this over .env
```

Comment:
The env example file shouldn't be removed in this PR.  If it does need to be removed do that in a seperate PR.

Reply:
oops, that was just me being lazy and renaming the env example